test/cmd/builds.sh

```diff
@@ -12,16 +12,64 @@ os::log::install_errexit
 url=":${API_PORT:-8443}"
 project="$(oc project -q)"
 
+# expect_build_output_spec runs `oc new-build` with given arguments and expects
+# a build config with a given name to be created with a given build output spec
+function os::cmd::builds::expect_build_output_spec() {
+	if [[ $# -ne 3 ]]; then echo "os::cmd::builds::expect_build_output_spec expects three argumments, got $#"; fi
+	local newbuild_args="$1"
+	local expected_buildconfig_name="$2"
+	local expected_output_spec="$3"
+
+	local template='{{with .spec.output.to}}{{.kind}} {{.name}}{{end}}'
+	local get_output_spec_cmd="oc get bc/${expected_buildconfig_name} --template '${template}'"
+
+	os::cmd::expect_success 'oc delete all --all'
```

Comment:
This line looks like something this function shouldn't be doing - I wouldn't expect this side effect from the name.